apps/dashboard/app/(app)/apis/_components/create-api-button.tsx

```diff
@@ -64,7 +64,7 @@ export const CreateApiButton = ({
     <>
       <Dialog open={open} onOpenChange={(o) => setOpen(o)}>
         <DialogTrigger asChild>
-          <Button variant="primary" {...rest}>
+          <Button variant="primary" {...rest} color="default">
```

Comment:
shouldn't the default color be the default value?

Reply:
That `{...rest}` is messing up the typescript check I gotta force the default color.